src/app.css

```diff
@@ -36,4 +33,31 @@
   text-decoration: none;
   font-weight: 400;
   color: inherit;
-}
\ No newline at end of file
+}
+
+.Harris {
+  display: none;
+  position: absolute;
+  width: 80%;
+  max-width: 400px;
```

Comment:
Google Analytics says ~50% of the mobile devices viewing the site have a viewport smaller than 400px. Not sure how much this matters, but it'd be nice to remain as "mobile friendly" as possible.

NOTE: I haven't checked any of this on my phone :P